CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+### 3.34
```

Comment:
Generally do not put an actual version. Like in the core, you're never really sure when a change is going to land.